test/minimalParams.ts

```diff
@@ -9,6 +9,8 @@ import {
   ITransferParams,
   TRANSACTION_TYPE, ISponsorParams, ICancelSponsorParams
 } from '../src/transactions'
+import {IAssociationParamsV3} from '../build/tmp/dist/transactions'
```

Comment:
these should import from `src` not `dist`. otherwise it can generate confusion/false results if running tests without building

Reply:
Yeah, sometimes my IDE wants other things than I want myself ;)